python/glow/gwas/lin_reg.py

```diff
@@ -117,6 +113,7 @@ def linear_regression(genotype_df: DataFrame,
     Y = phenotype_df.to_numpy(dt, copy=True)
     Y_mask = (~np.isnan(Y)).astype(dt)
     Y = np.nan_to_num(Y, copy=False)
+    Y_raw_nan_filled = np.nan_to_num(Y, copy=True) if verbose_output else None
```

Comment:
`Y` was already `nan`-filled; can we use `np.copy(Y)` [instead](https://numpy.org/doc/stable/reference/generated/numpy.copy.html)? Given this may also be `None`, it would also be clearer to name this variable something like `Y_for_verbose_output`.